pkg/tsdb/elasticsearch/time_series_query.go

```diff
@@ -72,23 +72,45 @@ func (e *timeSeriesQuery) processQuery(q *Query, ms *es.MultiSearchRequestBuilde
 	b.Size(0)
 	filters := b.Query().Bool().Filter()
 	filters.AddDateRangeFilter(e.client.GetTimeField(), to, from, es.DateFormatEpochMS)
-
-	if q.RawQuery != "" {
-		filters.AddQueryStringFilter(q.RawQuery, true)
-	}
+	filters.AddQueryStringFilter(q.RawQuery, true)
 
 	if len(q.BucketAggs) == 0 {
```

Comment:
The plan is to refactor this to whole `processQuery` function, but in this PR, I wanted to do as little changes as possible, so it is easier to review. Then, when we have all tests passing, I'll work on refactor.